src/css/components/_adaptive.scss

```diff
@@ -28,11 +33,6 @@
     .vjs-duration,
     .vjs-remaining-time,
     .vjs-playback-rate,
-    .vjs-chapters-button,
-    .vjs-descriptions-button,
-    .vjs-captions-button,
-    .vjs-subtitles-button,
-    .vjs-audio-button,
```

Comment:
Instead of simply showing the subtitles/captions button, show all track selection buttons, by default. This seemed more prudent in the light of accessibility concerns.